src/components/PhotoPage/FAQSection.js

```diff
@@ -0,0 +1,181 @@
+import MuiAccordionDetails from '@material-ui/core/AccordionDetails';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+
+import Add from '@material-ui/icons/Add';
+import Remove from '@material-ui/icons/Remove';
+
+// Stolen from HOTH website, yoink
+
+const Accordion = withStyles(theme => ({
+	root: {
+		backgroundColor: theme.palette.grey[50],
+		boxShadow: 'none',
+		borderColor: theme.palette.grey[400],
+		borderTopWidth: 2,
+		borderTopStyle: 'solid',
+		'&:last-child': {
+			borderBottomWidth: 2,
+			borderBottomStyle: 'solid'
+		},
+		'&::before': {
+			display: 'none'
+		},
+		'&$expanded': {
+			margin: 0
+		}
+	},
+	expanded: {}
+}))(MuiAccordion);
+
+const AccordionSummary = withStyles(theme => ({
+	root: {
+		minHeight: 'auto',
+		'&$expanded': {
+			minHeight: 'auto'
+		},
+		[theme.breakpoints.down('xs')]: {
+			paddingLeft: theme.spacing(1),
+			paddingRight: theme.spacing(1)
+		}
+	},
+	content: {
+		margin: '12px 0',
+		'&$expanded': {
+			margin: '12px 0'
+		}
+	},
+	expanded: {}
+}))(MuiAccordionSummary);
+
+const AccordionDetails = withStyles(theme => ({
+	root: {
+		paddingBottom: theme.spacing(6),
+		[theme.breakpoints.down('xs')]: {
+			paddingBottom: theme.spacing(4),
+			paddingLeft: theme.spacing(1),
+			paddingRight: theme.spacing(1)
+		}
+	}
+}))(MuiAccordionDetails);
+
+const useStyles = makeStyles(theme => ({
+	question: {
+		fontWeight: theme.typography.fontWeightMedium,
+		color: theme.palette.secondary.main,
+		fontFamily: theme.typography.fontFamily
+	}
+}));
+
+function AccordionQA({ index, question, answer }) {
+	const [expanded, setExpanded] = useState(index === 0);
+	const classes = useStyles();
+	const panelName = 'faqPanel' + index;
+	const theme = useTheme();
+
+	return (
+		<Accordion
+			key={panelName}
+			square
+			expanded={expanded}
+			onChange={() => setExpanded(e => !e)}>
+			<AccordionSummary
+				expandIcon={expanded ? <Remove /> : <Add />}
+				aria-controls={panelName + '-content'}
+				id={panelName + '-header'}>
+				<Typography variant='body1' className={classes.question}>
+					{question}
+				</Typography>
+			</AccordionSummary>
+			<AccordionDetails>
+				<Typography variant='body1' style={{ maxWidth: theme.breakpoints.values.md * 0.8 }}>
+					{answer}
+				</Typography>
+			</AccordionDetails>
+		</Accordion>
+	);
+}
+
+AccordionQA.propTypes = {
+	question: PropTypes.node.isRequired,
+	answer: PropTypes.node.isRequired,
+	index: PropTypes.number.isRequired
+};
+
+function FAQSection() {
+	const faqs = [
+		{
+			question: `Where do I find the ACM youtube channel?`,
+			answer:
+				<>
+					We post all recordings of our workshops on the ACM youtube channel, which can be found <Link href="https://www.youtube.com/c/ACMUCLA/videos">here</Link>!
+				</>
+		},
+		{
+			question: `How do I participate?`,
+			answer:
+				<>
+					We recommend attending workshops and events (such as HOTH or Hack Kitchen)
+					, joining a Hack Fam, or asking us questions in the ACM discord!
```

Comment:
```suggestion
					We recommend attending workshops and events (such as HOTH or Hack
					Kitchen), joining a Hack Fam, or asking us questions in the ACM discord!
```

Currently there's a space rendered before the comma.

Reply:
We might not always be holding events Hack Fam, HOTH, or Hack Kitchen every quarter. To keep maintaining this simple we should just saying something like "check out our events page to see what events we have this quarter! Everyone is able to join!" Be sure to emphasize that _anyone_ can join. That's something we always try to let people know.